### PR TITLE
Chore: Enable functional tests

### PR DESCRIPTION
**Pull request by jeremypress**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @jeremypress has signed the CLA. Thanks for the pull request!

File: package.json [hunk 1]
```diff
@@ -98,7 +98,7 @@
     "build": "yarn run clean && yarn run build-rb && yarn run lint && yarn run dev",
     "build-ci": "./node_modules/.bin/webpack --progress --colors --config build/webpack.config.js",
     "build-rb": "mojito-rb-gen -s src/i18n -o src/i18n/json -b en-US.properties",
-    "ci": "yarn run clean && yarn run build-rb && yarn run lint && yarn run test && yarn run build-ci",
+    "ci": "yarn run clean && yarn run build-rb && yarn run build-ci && yarn run lint && yarn run test && yarn run functional-tests-ci",
```
Comment (jeremypress) [inline package.json]: I believe the reason we were timing out was due to us building right before running the tests. This updates the `dist` folder which is being served by the python SimpleHTTPServer. By moving the build to before the unit tests, it gives the simple server plenty of time to update before running our functional tests.

File: .travis.yml [hunk 2]
```diff
@@ -1,5 +1,13 @@
 language: node_js
 node_js:
 - '6'
+addons:
+  sauce_connect:
+    username: box-content-preview
+  jwt:
+    secure: EPjmiionirVVkbFbp9r4DjMLMcfmdj9Sq8UiMcIGGf5gQk0cV5sgJpDHUDlKhFL3na8FkYf27qadaaJHG9Lr7o1V6IN1tsLamD9kdQjeG1pUSZbyMoMbja9i8CY0/MnQPaPL8UBXbcPrJoBwPNp6PphZ1PtBW68oXale3Xdfd7JMyTtm8f6G40pxpDl50dKI2VZHKKrfWakFkK2DMM73vJScxCrlyJEDc+wSJi2Mr/lrZc/QYy3ug3BGq1S6vV6wsOCk/4Jx+W1hvqK0QBgIytv1XzypqDFJ0OiTiOccsbwvEUHOepQl0EQxBZtM9PO//SBMFc2TqAhldta37JFrEcOEtqrsHwqVELzRMJm9lDSSD+TSj/tri6Kevy311gboZsJh2T3Znkq2h5Zpx8ImbgZCKvQyn8bTSp9MOkb9bGt1tUkZ1LEk3Pj0PUEjFZMnAsT/fLg2EGdXRQWb1W/ZyknoNPfvsU5Po8obQ/AOlKMqaMM7YiGQWruL7Esml0XaiL9H1TSUwmtyAnAU+Q1eYv7II7SnQmVxcgILX7D7nAO82/jP2f12fszEOlgivLr0AS9iAMLcNsrTfSO2p60MESmc/2ZVGjIagNEaGohfb9AI5d0tezU6+GWijuRijVqn1x2rLbSXOV7c3G9yrJpGTru9WD3n/ekgVvg/0nanBpU=
```
Comment (JustinHoldstock) [inline .travis.yml]: Are we okay putting this auth token in here?

Comment (jeremypress) [reply .travis.yml]: This is an encrypted JWT thing so its secure https://docs.travis-ci.com/user/jwt

**Pull request closed without merging**